nemo/collections/nlp/modules/common/megatron/adapters/mcore_mixins.py

```diff
@@ -256,9 +256,11 @@ def mcore_register_adapters(self):
 
     def forward(self, hidden_states):
         # [s, b, 4 * h/p]
-        linear_fc1_output, bias_parallel = self.linear_fc1(hidden_states)
-
-        intermediate_parallel, layernorm_output = linear_fc1_output
+        if self.linear_fc1.te_return_bias:
```

Comment:
no longer support non mcore path?

Reply:
this file `mcore_mixins.py` is only for mcore models